docs/operations.rst

```diff
@@ -0,0 +1,350 @@
+  #. The web and flapp servers which serves the website to users and implements signup.
+
+The volumes must be provisioned manually.
+Make sure they are in the same availability zone as the cluster
+(as defined in the ``kops create cluster`` command when it was created).
+Create a volume for the Docker registry.
+Make it around ten times the size of the site's infrastructure image [#volumesize]::
+
+  aws ec2 create-volume --availability-zone us-east-1a --size 8
+
+Take note of the ``VolumeId`` of the created volume.
+Edit ``k8s/registry.yaml``.
+Put the ``VolumeId`` into the ``volumeID`` field of the ``leastauthority-tweaks-kube-registry-pv`` resource.
+
+Repeat this procedure to create volumes for the web and flapp servers.
+These servers write human-readable logs and signup information to the volumes.
+One GiB each is most likely sufficient.
+Put the ``VolumeId`` values for these volumes into ``infrastructure.yaml``,
+in the ``infrastructure-web-pv`` and ``infrastructure-flapp-pv`` resources.
+
+You can now deploy the registry and web and flapp servers::
+
+  kubectl create -f "k8s/"
+
+Now build the Docker images for the site::
+
+  docker/build.sh
+
+Then set up a proxy to the private Docker registry.
+First, find the name of the registry's pod::
+
+  x=($(kubectl --namespace leastauthority-tweaks -o json get pods | jq -r .items[0].metadata.name))
+  name=${x[0]}
+  port=${x[1]}
+
+Then forward the port (stays in the foreground)::
+
+  kubectl port-forward $name $port
+
+Then tag the images so they can be pushed to the registry::
+
+  docker tag leastauthority.com/web 127.0.0.1:$port/leastauthority.com/web
+  docker tag leastauthority.com/flapp 127.0.0.1:$port/leastauthority.com/flapp
+
+*Then* push the images::
+
+  docker push 127.0.0.1:$port/leastauthority.com/web 127.0.0.1:$port/leastauthority.com/flapp
+
+At this point, you are almost done.
+Most likely, Kubernetes tried to start things up already and failed because the images were missing.
+This is okay.
+It will keep trying until things succeed.
+You can monitor the progress via the dashboard web interface or::
+
+  kubectl get pods --selector provider=LeastAuthority
+
+After the pods are running, you can use the AWS-provided LoadBalancer endpoint to access the service.
+Look it up using::
+
+  kubectl describe services --selector provider=LeastAuthority
+
+The ``LoadBalancer Ingress`` line should look like a domain name.
+Hit it via HTTPS.
+
+If you want, you can now create a CNAME for that domain to make the site accessible more easily.
+
+You're done.
+
+React to Increased Load
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If the Kubernetes cluster is suffering due to high load,
+it may be possible to improve the matter by adding additional Kubernetes worker nodes.
+
+.. attention::
+   The web server and signup processes do not support any kind of horizontal scale-out at this time.
+   Their performance will not be improved by adding additional worker nodes.
```

Comment:
Ah, interesting.  I assumed it would not currently be safe to have multiple consumers.  I don't know much about flappserver yet.  I'll have to learn some more about it before I make more changes in that area, I think.  It would be neat if it already had these kind of multi-reader queue features.

(Although I'm also going to vastly cheapen signup provisioning by switching to containers, soon.)